### PR TITLE
Add subscripts for `? stack`

### PR DESCRIPTION
**Pull request by JoaoFelipe3**
Added a subscript for `? stack`, which does the same amount of `⸮ trace`s, but displays as a subscript for `? stack`, of course.

I also changed `dump`'s documentation a tiny bit.

Hope you like this!

### CODE REVIEW

File: src/primitive/defs.rs [hunk 1]
```diff
@@ -3336,7 +3335,7 @@ macro_rules! impl_primitive {
             UndoReverse(usize),
             UndoRotate(usize),
             ReduceDepth(usize),
-            TraceN(usize, bool),
+            TraceN(usize, bool, bool),
```
Comment (kaikalii) [inline src/primitive/defs.rs:3338]: With 2 bools, this should be made a struct variant.

File: src/primitive/defs.rs [hunk 2]
```diff
@@ -2956,9 +2956,8 @@ primitive!(
     /// ex: [1 5 2 9 11 0 7 12 8 3]
     ///   : ▽×⸮≥5:⸮≤10..
     (1, Trace, Debug, ("trace", '⸮'), Impure),
-    /// Debug print all the values currently on stack without popping them
+    /// Debug print all stack values after applying a function to them without popping them
```
Comment (kaikalii) [inline src/primitive/defs.rs:2959]: I prefer these short descriptions to be shorter, even if they aren't super specific.

File: src/primitive/defs.rs [hunk 3]
```diff
@@ -2956,7 +2956,7 @@ primitive!(
     /// ex: [1 5 2 9 11 0 7 12 8 3]
     ///   : ▽×⸮≥5:⸮≤10..
     (1, Trace, Debug, ("trace", '⸮'), Impure),
-    /// Debug print all stack values after applying a function to them without popping them
+    /// Preprocess and print all stack values without popping them
```
Comment (JoaoFelipe3) [inline src/primitive/defs.rs:2959]: Does this look good to you?

**Comment by kaikalii** [comment]
I'm happy to merge this once checks pass.

**Pull request merged**